docs/user-guide/kubernetes.md

```diff
@@ -330,6 +330,59 @@ echo "$(minikube ip) traefik-ui.minikube" | sudo tee -a /etc/hosts
 
 We should now be able to visit [traefik-ui.minikube](http://traefik-ui.minikube) in the browser and view the Træfik Web UI.
 
+## Basic Authentication
+Is possible to add additional authentication annotations in the Ingress rule. The source of the authentication is a secret that contains usernames and passwords inside the the key auth.
+
+* Basic authentication only
+* Realm not configurable; only traefikdefault
+* Secret is in same namespace as ingress rule
+* Secret must contain only single file
+
+#### Creating the Secret
+
+1. `htpasswd -c ./auth myusername`
+
+   You will be prompted for a password which you will have to enter twice. htpasswd will create
+   a file with the following...
+   ```
+   ~$ cat auth
+   myusername:$apr1$78Jyn/1K$ERHKVRPPlzAX8eBtLuvRZ0
+   ```
+
+2. `kubectl --namespace=monitoring create secret generic mysecret --from-file auth`
```

Comment:
A quick sentence here that we are creating a secret containing the htpasswd file in a namespace called monitoring?